flow/api/middleware/middleware.go

```diff
@@ -0,0 +1,27 @@
+package middleware
+
+import "net/http"
+
+// CORS middleware for localhost environments
+func CorsMiddleware() func(http.Handler) http.Handler {
```

Comment:
maybe just to be explicit?

```suggestion
func CorsLocalhostMiddleware() func(http.Handler) http.Handler {
```